com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -919,8 +919,7 @@ internal void NetworkGetBehaviourVariable(UpdateCommand updateCommand, out Netwo
 
         internal int NetworkSendMessage(SnapshotDataMessage message, ulong clientId)
         {
-            m_NetworkManager.SendMessage(ref message, NetworkDelivery.ReliableFragmentedSequenced, clientId);
-
+            m_NetworkManager.SendPreSerializedMessage(message.WriteBuffer, TotalBufferMemory, ref message, NetworkDelivery.Unreliable, clientId);
```

Comment:
This simply calls a bit further in, after tmpSerializer, because we happen to have such a FastBufferWriter ready.